src/components/vm/overview/watchdog.jsx

```diff
@@ -37,7 +37,7 @@ import { WATCHDOG_INFO_MESSAGE } from './helpers.jsx';
 
 const _ = cockpit.gettext;
 
-const SUPPORTEDACTIONS = ["reset", "poweroff", "inject-nmi", "pause"];
+const SUPPORTEDACTIONS = ["reset", "poweroff", "inject-nmi", "pause", "none"];
```

Comment:
I see what this does in the test. So it is related to permanent watchdogs. However, this would/could still make sense as a separate commit (including the pixel change), if the "normal" watchdogs can have a "none" action as well? This also needs to gain a translation in helper.js's `watchdogAction` "human string" table.

Reply:
> This also needs to gain a translation in helper.js's watchdogAction "human string" table.

That's already there, afaics.

Reply:
I have split out the "none" action.

Reply:
I was mentioning the translation because the test does

>  b.wait_in_text("#vm-subVmTest1-watchdog-state", "none")

which looks wrong -- all other states are capitalized. Indeed I see the entry now:

    none: _("Do nothing"),

but then something is wrong.